pages/create-issue.tsx

```diff
@@ -45,14 +47,18 @@ export default function PageCreateIssue() {
 
   }
 
+  const title_and_description_validation = () => {
```

Comment:
please, follow naming conventions of with cammelCase: `titleAndDescriptionValidation` but that's not really what this function does, so `titleAndIssueExist` should be more telling of what this function _actually_ does

Reply:
Ok, done.